upf/test/test_upf.py

```diff
@@ -946,6 +958,7 @@ class TestPGWBase(PFCPHelper):
 
     CLIENT_TEID = 1
     UNODE_TEID = 1127415596
+    teid = UNODE_TEID
```

Comment:
Is this value modified somewhere in the test? If so, it must be set in `setUp()`, for example, or in the test methods.
Otherwise, it introduces dependency upon the ordering of the test run